src/main/java/org/cprover/CProverString.java

```diff
@@ -388,4 +388,26 @@ public static StringBuffer insert(
         return CProver.nondetWithoutNullForNotModelled();
     }
 
+    /**
+     * Converts an array of characters to a string. This uses a loop and
+     * therefore in test-generation the {@code count} will be limited by the
+     * unwind parameter.
+     * This assumes {@code value} is not null, its length is greater or equal to
+     * {@code offset + count} and {@code offset} and {@code count} are positive.
+     *
+     * @param value  Array of characters which is the source to copy from
+     * @param offset Index in {@code value} of the first character to copy
+     * @param count  Number of characters to copy
```

Comment:
We don't usually start the parameter description with an upper-case char.

Reply:
Hm. True, lower case seems standard in javadoc.